src/handler/refreshworker.h

```diff
@@ -27,18 +27,20 @@
 #include <QByteArray>
 #include <QHash>
 #include <QSet>
-#include "deferred.h"
 #include <boost/signals2.hpp>
+#include "deferred.h"
+#include "zrpcrequest.h"
 
 using Connection = boost::signals2::scoped_connection;
 
-class ZrpcRequest;
 class ZrpcManager;
 class StatsManager;
 class WsSession;
 
 class RefreshWorker : public Deferred
 {
+	Q_OBJECT
```

Comment:
Technically this should be here since `Deferred` inherits from `QObject`, but probably not for much longer.